### PR TITLE
adding Finland text

### PR DESCRIPTION
**Pull request by samumantha**
Moi,

Here a bit of text which could be the beginning of a Finland page (Richard suggested to do so in zulip)
Please feel free to edit the text as you think appropriate. If you have ideas you can also suggest here, and I can write something more. Since I am very new here, not that creative yet ;)

missing: how to join the coffeebreak, since its probably not a good idea to publicly post the zoom link without any attendee control. Maybe an email to @rantahar ?

Also I was not sure where to add a link to this (maybe a dropdown in the navbar under community?), so its not linked anywhere yet.

Personally, I think a few faces/stories would be nice to add here.Would make the whole thing more approachable in my opinion. What do you think? 

PS: Also my very first pull request outside of a course, hope everything went well

### CODE REVIEW

**Comment by rkdarst** [comment]
Thanks!

As for linking from the top bar, I think it would be in _includes/navbar.html, but I don't know exactly how it works.  Do we add another sub-list list, or what?  Hopefully someone else can make a suggestion, if not I might try to find a site to copy from (or really, I would just link from the community page but depends on how interested you are in this project).  I found [this](https://github.com/coderefinery/coderefinery.org/blob/gh-pages/_includes/navigation.html), but it'll take some translation and may not be the same system as we use here.

There is no 'Gemfile' in here, so the `bundler` thing I suggested won't work.  But I installed Jekyll through my OS's package manager, and then `jekyll build` produced it in `_site`.

I personally would put the zoom link right in the page.  We are small, it's easy to change later if trolls do find it.  We've had our public "scicomp garage" link on a web page for months, and nothing has ever happened.  Of course it could and we are prepared for that, but my hypothesiss is to make the link look as old and boring as possible, and not have exact dates listed so that it can't be searched.

Thanks again!

**Comment by bast** [comment]
I will add a Gemfile and do the navbar part. If you are happy with the text, let's merge and I adjust the stuff around it and ping you in my follow-up pull request.

File: countries/finland.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ for research by offering workshops and a platform for information exchange in Finland.
+
+To discuss matters concerning upcoming workshops in Finland, and also to get to know each other better, we organize a **weekly virtual coffee break**, 
+see below. Everyone with an interest in the topic is warmly welcome to join :)
+
+## Join the discussions
+
+### Chat
+
+Discuss with other RSEs on the [CodeRefinery chat](https://coderefinery.zulipchat.com) (nordic-rse stream).
```
Comment (rkdarst) [inline countries/finland.md:18]:
```suggestion
Discuss with other RSEs on the [CodeRefinery chat](https://coderefinery.zulipchat.com) (Join the #finland and possibly #nordic-rse streams using the gear icon by the steams list on the left side).
```

**Comment by bast** [comment]
Remaining issues tracked in #50 and #51.

**Comment by bast** [comment]
I finally fixed the navbar. Working on the rest of the page ...

**Pull request merged**